source/security/index.html.md.erb

```diff
@@ -168,5 +168,5 @@ clients and to provide secure connections.
 You must use HTTPS for all direct communication between your service and
 GOV.UK Pay.
 
-Return URLs for live services using GOV.UK Pay must use HTTPS, but [you can use HTTP for return
-URLs with test accounts](/testing_govuk_pay/#testing-gov-uk-pay).
+Return URLs for live services using GOV.UK Pay must use HTTPS, but you can use HTTP for return
+URLs with test accounts.
```

Comment:
I think that somewhere we need to define what we mean by "test account". I'd suggest adding a heading to the Quick Start guide after "Before you start" and before "The GOV.UK Pay API", saying something like:

h2. Types of account

GOV.UK Pay offers two types of account:

- a *test account* is a simulation of our system which you can use to explore and understand how GOV.UK Pay works, and to help develop your integration. 
- a *live account* is needed to process live payments (see going live)

Reply:
Good idea, thanks David - I've added a version of this on the Quick start page in https://github.com/alphagov/pay-tech-docs/commit/041acc8d5a90525b0006afc2e2f2157fdd086b5e.